lib/keychain/keychain.rb

```diff
@@ -102,6 +136,37 @@ def generic_passwords
       Scope.new(Sec::Classes::GENERIC, self)
     end
 
+    # Imports item from string or file to this keychain
+    #
+    # @param [String, File] input String or File with raw data to import
+    # @param [Array <String>] trusted_apps List of applications which will be
+    # permitted to access imported items
+    def import(input, trusted_apps=[])
+      input = input.read if input.is_a? File
+
+      # Create <CFArray> with references to <SecTrustedApplication> objects
+      app_array = CF::Array.mutable.release_on_gc
+      trusted_apps.each do |app|
```

Comment:
I've tried to operate with ruby array, but that I should to add a class for `SecTrustedApplication`, which instances are returned by `SecTrustedApplicationCreateFromPath`. So, `.to_cf` is failed.
Should I add `SecTrustedApplication` class here?